5-causation/01-defining-causality/slides.qmd

```diff
@@ -26,6 +26,7 @@ execute:
 
 -  Announcements
 -  Concept Questions
+-  Course Evaluations
```

Comment:
Was the verdict to reserve class time for this? It sounded to me like Jeremy thought we should nix this give the extra CQ.

Reply:
Good catch, I added this at the beginning of the instructor meeting but meant to remove it.  Just re-pushed with this removed.